serverless.yml

```diff
@@ -193,6 +195,7 @@ resources:
               NonKeyAttributes:
                 - primaryVrm
                 - secondaryVrms
+                - systemNumber
```

Comment:
should this be a part of both key schema as well as non key attributes?

Reply:
I had the same question. I think is fine because it's a non-key attribute on `GlobalSecondaryIndexes` (hidden by the GitHub interface)

Reply:
Yeah, systemNumber is a Local key, but on the GSIs, they have their own index keys. These NonKeyAttributes effectively determine which fields we get back (apparently)